index.html

```diff
@@ -15,208 +15,416 @@
 </head>
 
 <body>
+
     <header>
         <div class="header container">
             <a class="header-logo" href="/">My Logo</a>
             <nav>
                 <i class="header-icon fa-solid fa-bars"></i>
+                <div class="nav-links">
+                    <a href="">Portfolio</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                </div>
             </nav>
         </div>
     </header>
+    <div class="first-background">
+        <section class="section-info">
+            <div class="background">
+                <div class="back-images">
+                    <img src="./img/desktop-version/Rectangle6.svg" alt="" class="image1">
+                    <img src="./img/desktop-version/Rectangle4.svg" alt="" class="image2">
+                    <img src="./img/desktop-version/Ellipse11.svg" alt="" class="image3">
+                    <img src="./img/desktop-version/Rectangle7.svg" alt="" class="image4">
+                    <img src="./img/desktop-version/Ellipse5.svg" alt="" class="image5">
+                    <img src="./img/desktop-version/Ellipse3.svg" alt="" class="image6">
+                    <img src="./img/desktop-version/Ellipse12.svg" alt="" class="image7">
+                    <img src="./img/desktop-version/Ellipse13.svg" alt="" class="image8">
+                    <img src="./img/desktop-version/Ellipse4.svg" alt="" class="image9">
+                    <img src="./img/desktop-version/Rectangle5.svg" alt="" class="image10">
+                    <img src="./img/desktop-version/Ellipse8.svg" alt="" class="image11">
+                    <img src="./img/desktop-version/Ellipse8.svg" alt="" class="image12">
+                    <img src="./img/desktop-version/Ellipse8.svg" alt="" class="image13">
+                </div>
```

Comment:
- [ ] Kindly remove the div of images and instead use background-image in CSS you can check this [link](https://www.w3schools.com/css/css3_backgrounds.asp) if you want to add multiple background images.
- [ ] As you can see that it has made the design in the mobile version doesn't look responsive.
![Screenshot from 2022-04-28 23-12-22](https://user-images.githubusercontent.com/33752735/165846751-24740fa6-2964-41bb-ae62-ed56319acc9a.png)